src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ObjectFormatting/Formatters/DebuggerDisplay/ExpressionBinder.cs

```diff
@@ -26,7 +27,7 @@ internal record struct ExpressionEvaluator(BoundEvaluator Evaluate, Type ReturnT
 
             if (debuggerDisplay.Expressions.Count == 0)
             {
-                return (_, _) => debuggerDisplay.FormatString;
+                return (_, _) => new() { FormattedValue = debuggerDisplay.FormatString, Flags = ParameterEvaluationFlags.None };
```

Comment:
There's only one caller, so it should be able to take the formatted string and turn it into a `ObjectFormatterFunc`.

Reply:
I can't return a `string?` because the `ObjectFormatterFunc` takes in a `formatSpecifier` and the `object` as input and that's used to construct the final string (see lines 52 and 62).
I could return a new delegate type that takes an `object` and `FormatSpecifier` as input and returns a `string` instead of `ObjectFormatterResult`, but that seems like a very specific type that's only used in a single place.
What do you think?

Reply:
I updated the code on Joe's suggestion to default the eval result to `Success`, so there's no need to always set it.